packages/realm-react/CHANGELOG.md

```diff
@@ -7,8 +7,7 @@
 * None
 
 ### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-js/issues/????), since v?.?.?)
-* None
+* Fix for useObject not updating when using previously used primary key. ([#5620]https://github.com/realm/realm-js/issues/5620) thanks @RS1-Project)
```

Comment:
```suggestion
* Fix for `useObject` not updating when using previously used primary key. ([#5620]https://github.com/realm/realm-js/issues/5620) thanks @RS1-Project)
```

Reply:
Do we know when the defect was introduced? If so, please add `since vX.Y.Z`

Reply:
Updated!